roles/systemd_hostnamed/tasks/systemd_service.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Ensure {{ systemd_hostnamed_service_name }} systemd service
+- name: Ensure systemd service {{ systemd_hostnamed_service_name }}
```

Comment:
can you make a separate merge/pull request about lint fixes like this?

Reply:
I can but then your linting would not accept the pull request.